mathe.sty

```diff
@@ -121,7 +121,7 @@
 \newcommand{\gleichwegen}[1]{\ensuremath{\DOTSB\;\stackrel{#1}{=}\;}}
 \newcommand{\da}{\mathrel{\mbox{$\coloneqq$}}}
 \newcommand{\ad}{\mathrel{\mbox{$\eqqcolon$}}}
-\newcommand{\dequizu}{\ensuremath{\mathrel{\vcentcolon\Longleftrightarrow}}}
+\newcommand{\dequizu}{\ \ensuremath{\mathrel{\vcentcolon\Longleftrightarrow}}\ }
```

Comment:
Hmm, sollte `mathrel` nicht für korrektes Spacing sorgen?

Reply:
Nicht wenn das Symbol in der Form `a \dequizu b` ohne Math Mode verwendet wird.

Reply:
Dann sollte das eben im math mode geschehen :-)